### PR TITLE
Adds super fancy image to readme

### PR DESCRIPTION
**Pull request by marcusnovotny**

### CODE REVIEW

**Comment by julienschmidt** [comment]
Please limit the screenshot on the graph that we provide

**Comment by julienschmidt** [comment]
Also please use markdown syntax without size values

**Comment by marcusnovotny** [comment]
Image switched. Gonna use the other one for our social media post

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # gotsentimental [![Build Status](https://travis-ci.org/Rostlab/JS16_ProjectD_Group4.svg?branch=develop)](https://travis-ci.org/Rostlab/JS16_ProjectD_Group4) [![npm](https://img.shields.io/npm/v/gotsentimental.svg)](https://www.npmjs.com/package/gotsentimental)
 GoT Twitter Sentiment Analysis
 
+![](http://i.imgur.com/DTjD3bt.png)
```
Comment (julienschmidt) [inline README.md]: There is a reason why the alt text is not optional :wink:

**Pull request merged**